### ISSUE TITLE
load_from = "../pretrain/epoch_24.pth", Can you provide the epoch 24.pth file

### ISSUE THREAD

**Issue by luoyq6** [Bohao-Lee/PDC#2]
Hello, load_from = ".. in your configuration file imted_ss_faster_rcnn_vit_base_2x_finetuning_10shot_coco.py /pretrain/epoch_24.pth", is the weight trained with imTED?

**Comment by Bohao-Lee** [comment]
Yes, our method(PDC) only applies to finetune stage. Hence, the base training stage is just followed imTED.
